applications/system-service/wlan.h

```diff
@@ -50,6 +62,31 @@ class Wlan : public QObject, public SysObject {
             return 0;
         }
     }
+    signed int rssi(){
+        QDBusMessage message = m_interface->call("SignalPoll");
```

Comment:
Should we handle the case where wpa_supplicant returns an error here? Something like this:
```suggestion
        QDBusMessage message = m_interface->call("SignalPoll");
        if (message.type() == QDBusMessage::ErrorMessage) {
            qWarning() << "SignalPoll error: " << message.errorMessage();
            return -100;
        }
```